cdci_data_analysis/flask_app/app.py

```diff
@@ -275,7 +275,9 @@ def run_analysis():
 def validate_schema(response):
     try:
         if dispatcher_strict_validate:
-            QueryOutJSON().load(response.json)
+            # TODO in case of download/js9 request a dedicated validaiton schema should be defined
+            if not response.is_streamed:
```

Comment:
separating-out the case of download file is a step for more clear response schema, why not